contributed_definitions/NXcoordinate_system_set.nxdl.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXcoordinate_system_set" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance">
+    <doc>
+         Container to hold different coordinate systems conventions.
+         
+         By default NeXus uses the McStas coordinate system, whose axis convention 
+         is so that the z axis is along the neutron propagation axis, 
+         the y axis is vertical up, and the x axis points left when looking along
+         the z-axis, completing a right-handed coordinate system.
+         Assuming the McStas coordinate system as the reference (indicated by &quot;.&quot;),
+         a specific experiment or simulation may defined further coordinate systems.
+         
+         It is the purpose of this base class to define these conventions and
+         offer a place to store mappings between coordinate systems.
+         
+         For each coordinate system one should use a set of NXtransformations to define:
+         * The axis conventions
+         * The affine transformations which bring each axis of this coordinate system
+         into registration with the McStas coordinate system.
+         * Equally affine transformations from the inverse mapping should be specified.
+         
+         As an example one may take an experiment or computer simulation where
+         there is a laboratory (lab) coordinate system, a sample/specimen coordinate
+         system, a crystal coordinate system, and additional coordinate systems,
+         which are eventually attached to components of the instrument.
+         
+         If no additional transformation is specified in this group or if an
+         instance of an NXcoordinate_system_set is absent it should be assumed the
+         McStas coordinate system is used.
+    </doc>
+    <group type="NXtransformations">
+        <doc>
+             A group of transformations which specify:
```

Comment:
reST bullet list, needs blank lines before and after

Reply:
fixed